test/e2e/run_networking_test.go

```diff
@@ -541,4 +541,19 @@ var _ = Describe("Podman run networking", func() {
 		Expect(run.ExitCode()).To(BeZero())
 		Expect(run.OutputToString()).To(ContainSubstring(ipAddr))
 	})
+
+	It("podman run with new:pod and static-ip", func() {
+		SkipIfRemote()
+		SkipIfRootless()
+		netName := "podmantestnetwork2"
+		ipAddr := "10.20.40.128"
+		create := podmanTest.Podman([]string{"network", "create", "--subnet", "10.20.40.0/24", netName})
```

Comment:
Please remove this at the end of the test - these are not namespaces or automatically removed at the end of each test.

Also, a different subnet that is less likely to collide with other tests would be a good idea.

Reply:
What subnet would you recommend? something in 172.16... or 192.168... ?

Reply:
10.0.0.0 is fine, we use it elsewhere. We do just some 10.20.x.x subnets elsewhere, though. Maybe 10.25.40.0?

Reply:
There is a test directly above which uses 10.20.30.0/24 and does not remove the network. Should I also change this test while I'm at this?

Reply:
Yes please!